scalableminds/webknossos
# Remove D3 Dependency

With the recently introduced TimeTracking view, we integrated Google Charts in wk. Therefore, we should get rid of all D3 / C3 charts. In this PR I removed the per user dashboard for monthly tracing times since it has been superceded by the new TimeTracking view. Additionally, I replaced the last and only remaining chart on the statistics view with Google Charts.

### Mailable description of changes:
 - [Dashbard.TimeSummary] Removed the monthly working time overview from the dashboard. For fine grained time tracking insights, please use the new time tracking view.
- [Statistics] Updated Line Chart in Statistics View

### Steps to test:
- Go to Admin --> Users --> View Tracings. The Time tab should be gone.
- Go to Navbar --> Statistics. Line Chart should work and show times.
- Click on any data point in the line chart. The table below should update to show the top 5 user for that week.

### Issues:
- fixes #2193 

Google Line Charts Docs: 
https://developers.google.com/chart/interactive/docs/gallery/linechart#examples

------
